translations/introduction.tex/fr.po

```diff
@@ -79,3 +106,6 @@ msgid ""
 "  \\centering\n"
 "  \\includegraphics[width=\\linewidth, height=\\myspace, keepaspectratio]{\\art/hydra.png}"
 msgstr ""
+"[blanker]\n"
+"  \\centering\n"
+"  \\includegraphics[width=\\linewidth, height=\\myspace, keepaspectratio]{\\art/hydra.png}"
```

Comment:
```suggestion
```
If there's nothing to translate, you can skip it. `po4a` picks up the original in this case.